powerhome/playbook
# [PLAY-996] Body and Title Truncate Docs + Prevent Class Name Clash

**What does this PR do?**
- [Runway](https://nitro.powerhrg.com/runway/backlog_items/PLAY-996)
- Adds `truncate` prop docs for Body and Title kits
- Title kit: prevent CSS attribute selector clash between the `size` and `truncate` props by adding a hyphen to the truncate classnames
  - As an example, `size` has an attribute selector of `[class*=_1]` and `truncate` had a class name like `truncate_1`. This selector would inadvertently match `size`'s CSS to `truncate`.
  - TODO: We should update the kits + Nitro to use something like `[class*=_size_1]` to prevent future clashes

**Screenshots:**

**Body**
![Screenshot 2023-09-19 at 10 14 35 AM](https://github.com/powerhome/playbook/assets/47684670/c1a6bd97-5f98-447f-944c-744f61fe7431)

**Title**
![Screenshot 2023-09-19 at 10 14 16 AM](https://github.com/powerhome/playbook/assets/47684670/e086d9c9-de67-4ad1-b9e0-05cb6ed6d86a)

**How to test?**
1. Go to the Body and Title docs
2. See the new Truncate example
3. Ensure that Title sizes still behave correctly

#### Checklist:
